### PR TITLE
#35 Windows CI check

### PR DESCRIPTION
**Pull request by bms63**

### CODE REVIEW

**Comment by bms63** [comment]
Hi all,  

I got these CI checks to work for windows and mac OS to make sure the package builds on those machines.   Also, cleaned up the workflows a little bit and renamed some things.

@thomas-neitmann  will the FDA be able to use `envsetup` as it is not available on CRAN.  I had to install it from github in the CI check.  Unsure if this will be an issue for them once we send it over the fence?

**Comment by bms63** [comment]
Hi all,  Can I get an approval/review please - @robertdevine  @thomas-neitmann  @laxamanaj

**Review by robertdevine — APPROVED** [review]
Thank you, @bms63 

This will help during test phase and submission preparation for Pilot 3. R Infrastructure might also make note.

File: .github/workflows/r-cmd-check.yml [hunk 1]
```diff
@@ -34,6 +30,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
+         # - {os: windows-latest, r: '4.2', repos: 'https://cran.microsoft.com/snapshot/2022-11-01/'}
           - {os: ubuntu-20.04, r: '4.2', repos: 'https://cran.microsoft.com/snapshot/2022-11-01/'}
```
Comment (thomas-neitmann) [inline .github/workflows/r-cmd-check.yml:34]: Why don't you add the Windows check here instead of creating a new file?

Comment (thomas-neitmann) [reply .github/workflows/r-cmd-check.yml:34]: ALso can you upgrade the snapshot to `2022-12-14`?

Comment (bms63) [reply .github/workflows/r-cmd-check.yml:34]: Sure happy to upgrade.  The two weren't playing nice together (linux and windows) so I just split them out.  Definitely, a better way to do it , i.e. put them together, just didn't figure it out.

**Comment by robertdevine** [comment]
Thank you, @bms63. Excellent! 

@thomas-neitmann and @laxamanaj, R Infrastructure group will appreciate and Pilot 3 testers during test phase as well. Thank you.

**Pull request merged**